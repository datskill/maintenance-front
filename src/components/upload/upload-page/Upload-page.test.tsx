import * as React from "react";
import * as renderer from "react-test-renderer";
import UploadPage from "./Upload-page";

it("UploadPage renders correctly without maintenance", () => {
  const onMaintenance = false;
  const tree = renderer
    .create(<UploadPage onMaintenance={onMaintenance} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it("UploadPage renders correctly in maintenance", () => {
    const onMaintenance = true;
    const tree = renderer
      .create(<UploadPage onMaintenance={onMaintenance} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });