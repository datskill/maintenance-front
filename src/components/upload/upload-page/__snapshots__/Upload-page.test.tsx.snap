// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UploadPage renders correctly in maintenance 1`] = `
<div
  className="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column MuiGrid-align-items-xs-center MuiGrid-justify-xs-center"
>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-md-12"
  >
    <p
      className="styled-p-margin"
    >
      Déposer un fichier CSV
    </p>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-md-12"
  >
    <input
      accept=".csv, .xml"
      className="display-nothing"
      disabled={true}
      id="contained-button-file"
      multiple={true}
      onChange={[Function]}
      type="file"
    />
    <label
      htmlFor="contained-button-file"
    >
      <span
        aria-disabled={false}
        className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={0}
      >
        <span
          className="MuiButton-label"
        >
          <span
            className="MuiButton-startIcon MuiButton-iconSizeMedium"
          >
            <svg
              aria-hidden="true"
              className="MuiSvgIcon-root"
              focusable="false"
              role="presentation"
              viewBox="0 0 24 24"
            >
              <path
                d="M17 3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2V7l-4-4zm-5 16c-1.66 0-3-1.34-3-3s1.34-3 3-3 3 1.34 3 3-1.34 3-3 3zm3-10H5V5h10v4z"
              />
            </svg>
          </span>
          Déposer un fichier
        </span>
        <span
          className="MuiTouchRipple-root"
        />
      </span>
    </label>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-md-12"
  >
    <p>
      Les fichiers suivants sont importés :
    </p>
    <ul />
  </div>
  <div
    className="MuiGrid-root padding-button MuiGrid-item MuiGrid-grid-md-12"
  >
    <label
      aria-disabled={true}
      className="MuiButtonBase-root MuiButton-root MuiButton-contained Mui-disabled Mui-disabled"
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      role="button"
      tabIndex={-1}
    >
      <span
        className="MuiButton-label"
      >
        Envoyer
      </span>
    </label>
  </div>
</div>
`;

exports[`UploadPage renders correctly without maintenance 1`] = `
<div
  className="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-column MuiGrid-align-items-xs-center MuiGrid-justify-xs-center"
>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-md-12"
  >
    <p
      className="styled-p-margin"
    >
      Déposer un fichier CSV
    </p>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-md-12"
  >
    <input
      accept=".csv, .xml"
      className="display-nothing"
      disabled={false}
      id="contained-button-file"
      multiple={true}
      onChange={[Function]}
      type="file"
    />
    <label
      htmlFor="contained-button-file"
    >
      <span
        aria-disabled={false}
        className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary"
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        role="button"
        tabIndex={0}
      >
        <span
          className="MuiButton-label"
        >
          <span
            className="MuiButton-startIcon MuiButton-iconSizeMedium"
          >
            <svg
              aria-hidden="true"
              className="MuiSvgIcon-root"
              focusable="false"
              role="presentation"
              viewBox="0 0 24 24"
            >
              <path
                d="M17 3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2V7l-4-4zm-5 16c-1.66 0-3-1.34-3-3s1.34-3 3-3 3 1.34 3 3-1.34 3-3 3zm3-10H5V5h10v4z"
              />
            </svg>
          </span>
          Déposer un fichier
        </span>
      </span>
    </label>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-md-12"
  >
    <p>
      Les fichiers suivants sont importés :
    </p>
    <ul />
  </div>
  <div
    className="MuiGrid-root padding-button MuiGrid-item MuiGrid-grid-md-12"
  >
    <label
      aria-disabled={true}
      className="MuiButtonBase-root MuiButton-root MuiButton-contained Mui-disabled Mui-disabled"
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      role="button"
      tabIndex={-1}
    >
      <span
        className="MuiButton-label"
      >
        Envoyer
      </span>
    </label>
  </div>
</div>
`;
